README.md

```diff
@@ -62,3 +62,13 @@ holepunch foo_bar --cidr=192.168.0.0/16 22 80
 # Also works with IPv6 ranges
 holepunch foo_bar --cidr=2001:882f::1/128 443
 ```
+
+Less verbose or quiet mode:
+
+```
+# Verbose mode
+holepunch foo_bar -v2 --cidr=192.168.0.0/16 22 80
+
+# Quiet mode
+holepunch foo_bar -v0 --cidr=2001:882f::1/128 443
```

Comment:
IMO this is overkill for how simple of a tool `holepunch` is. Using `-v 0` to disable messages with `-v 2` being the default feels a little off to me.

The intention of the messages that get printed currently wasn't debugging, but rather to make sure the user is confident they're making the correct changes. If any of the messages seem to be not useful, maybe we can remove or reword them instead.

I do like the idea of having a quiet mode, but I think my preference here would be to have two options: 

- print progress messages (default, no flag)
- don't print (maybe `-s, --silent` as in `curl`)

i.e. 

```
# Verbose output (default)
holepunch foo_bar --cidr=192.168.0.0/16 22 80

# Quiet output
holepunch foo_bar --silent --cidr=2001:882f::1/128 443
```

If we really need it, maybe with an additional `--debug` option to produce more information (possibly even enabling logging inside boto etc), but I'm not sure that's necessary.